src/ch/imedias/rsccfx/view/RsccSupportPresenter.java

```diff
@@ -61,45 +58,54 @@ public void initSize(Scene scene) {
   }
 
   /**
-   * Validates the token and displays a symbolic image.
-   *
-   * @param token the token to be validated.
-   * @return path to the image to display.
+   * Sets the token validation image depending on the validity of the token.
```

Comment:
Where does the value get validated? I excepted it in here, since I wouldn't search in "initBindings()" at all - also since it is at the end of the file.